test/test_translator.py

```diff
@@ -155,6 +218,23 @@ def test_get_road_direction():
 
     assert test_direction==valid_direction
 
+def test_get_road_direction_invalid_direction():
```

Comment:
We're running three separate tets in this method. Let's break those out into appropriate named methods and test each separately. For instance:
```
def test_get_road_direction_empty_coordinates():
    test_coordinates = []
    test_direction = icone_translator.get_road_direction(test_coordinates)
    expected_direction = None
    assert expected_direction == test_direction
```

Reply:
i changed the name test_get_road_direction_empty_coordinates(): and did break out into appropriate named methods